src/Streamer.cpp

```diff
@@ -8,6 +8,16 @@ std::chrono::milliseconds systemTime() {
   return std::chrono::duration_cast<std::chrono::milliseconds>(
       now.time_since_epoch());
 }
+bool stopTimeElapsed(std::uint64_t MessageTimestamp,
+                     std::chrono::milliseconds Stoptime) {
+  if (Stoptime.count() > 0 and
+      static_cast<std::int64_t>(MessageTimestamp) >
+          std::chrono::duration_cast<std::chrono::nanoseconds>(Stoptime)
+              .count()) {
+    return true;
+  }
+  return false;
```

Comment:
The if statement is superfluous, could just
```
return Stoptime.count() > 0 and
    static_cast<std::int64_t>(MessageTimestamp) >
        std::chrono::duration_cast<std::chrono::nanoseconds>(Stoptime)
            .count();
```

Reply:
Correct, fixed.